.travis.yml

```diff
@@ -1,14 +0,0 @@
-language: node_js
-node_js:
-  - "lts/*"
-before_install:
-  - wget http://www.imagemagick.org/download/ImageMagick.tar.gz
-  - tar xf ImageMagick.tar.gz
-  - cd ImageMagick*
```

Comment:
Can you confirm that the regression tests run correctly? (i.e., ImageMagick is already installed in ubuntu-latest.)

Reply:
I don't know what is correct. It least the tests are all green and don't fail. You can see here in my fork:
https://github.com/wassertim/vexflow/pull/1/checks?check_run_id=1488734596

Reply:
Would be nice if you could give me a hint what exactly to check.